src/browser/api/window.js

```diff
@@ -506,16 +499,6 @@ Window.create = function(id, opts) {
             _externalWindowEventAdapter = new ExternalWindowEventAdapter(browserWindow);
         }
 
-        let teardownListeners = () => {
-            // tear down any listeners...
-            _openListeners.forEach(unhook => {
```

Comment:
This **won't** clean 11 remaining events: blur, close, closed, focus, hide, maximize, minimize, responsive, restore, show, unresponsive